RegExp.make.js

```diff
@@ -306,7 +306,7 @@ RegExp.make = (function () {
    *    a start context, and RegExp source, and returns an end context.
    */
   function parseRegExpSource(eventHandler) {
-    const {
+    var {
```

Comment:
This is the same limitation of closure compiler.  It types `const`s in a separate pass from `var`s.